org.kde.haruna.yml

```diff
@@ -0,0 +1,181 @@
+---
+app-id: org.kde.haruna
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: haruna
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=pulseaudio'
+  - '--device=dri'
+  - '--filesystem=host'
```

Comment:
do you really host access?

Reply:
When a file is opened it adds files from the same folder to the playlist.
Users can define folders where for search for subtitles